loleaflet/src/control/Control.Menubar.js

```diff
@@ -751,26 +751,30 @@ L.Control.Menubar = L.Control.extend({
 				{uno: '.uno:PreviousTrackedChange'},
 				{uno: '.uno:NextTrackedChange'}
 			]},
-			{name: _('Page Setup'), id: 'pagesetup', type: 'action'},
-			{id: 'watermark', uno: '.uno:Watermark'},
 			{name: _UNO('.uno:ViewMenu', 'text'), id: 'view', type: 'menu', menu: [
 				{uno: '.uno:ControlCodes'},
 				{uno: '.uno:SpellOnline'},
 				{name: _UNO('.uno:ShowResolvedAnnotations', 'text'), id: 'showresolved', type: 'action', uno: '.uno:ShowResolvedAnnotations'},
 			]
 			},
+			{id: 'watermark', uno: '.uno:Watermark'},
+			{name: _('Page Setup'), id: 'pagesetup', type: 'action'},
+			{uno: '.uno:WordCountDialog'},
 			{name: _UNO('.uno:RunMacro'), id: 'runmacro', uno: '.uno:RunMacro'},
+			{name: _UNO('.uno:FullScreen', 'text'), id: 'fullscreen', type: 'action', mobileapp: false},
```

Comment:
@Andreas-Kainz this causes writer/mobile_wizard_state_spec.js cypress test to fail. I will move it to view submenu. (test checks if all items for the root level are visible on the screen without scrolling)

Reply:
oh sorry, thank's that you fix it.